data/actionsmap/yunohost.yml

```diff
@@ -436,7 +436,7 @@ app:
                     help: Custom name for the app
                 -a:
                     full: --args
-                    help: Serialize arguments for app installation
+                    help: Serialized arguments for app script (i.e. "domain=domain.tld&url_path=/path")
```

Comment:
Yep, you are right, we should call this variable `path`.
Fixed with https://github.com/YunoHost/yunohost/commit/1dd1ae483f1308a836af3cf4a8c9403ba26a591e.